algorithms/kernel/kmeans/kmeans_container.h

```diff
@@ -113,14 +115,15 @@ services::Status DistributedContainer<step1Local, algorithmFPType, method, cpu>:
     a[0] = static_cast<NumericTable *>(input->get(data).get());
     a[1] = static_cast<NumericTable *>(input->get(inputCentroids).get());
 
-    const size_t nr = 5 + (par->assignFlag != 0);
+    const size_t isAssignments = par->resultsToEvaluate & computeAssignments || par->assignFlag;
```

Comment:
Why do we need 'assignFlag' still in interface2?

Reply:
We are required to support this parameter until we delete it.